app/tailwind.config.js

```diff
@@ -1,8 +1,14 @@
 /* eslint-disable unicorn/prefer-module */
 
+
 module.exports = {
+ 
   content: ['./pages/**/*.{js,ts,jsx,tsx}', './frontend/**/*.{js,ts,jsx,tsx}'],
   theme: {
+    fontFamily: {
+    'monospaced':'Montserrat' },
```

Comment:
The key should be: `mono` not `monospaced`

Also, we should keep the default values as backup: https://tailwindcss.com/docs/font-family#customizing-the-default-font

And try to install the font: https://github.com/fontsource/fontsource/tree/main/fonts/google/montserrat#readme